react/demo/src/daisyui/samples/slider/Default.route.tsx

```diff
@@ -0,0 +1,16 @@
+import {useState} from 'react';
+import {Slider} from './Slider';
+
+const TailwindDemo = () => {
```

Comment:
Rename to DaisyUIDemo or smth more meaningful, TailwindDemo is not a great name.